misc/scripts/search.sh

```diff
@@ -57,7 +57,7 @@ if [[ $PACKAGE == *@* ]]; then
 			if [[ "$LEN" -eq 0 ]]; then
 				fancy_message warn "There is no package with the name $IRed${PACKAGE%%@*}$NC in the repo $CYAN$REPONAME$NC"
 				error_log 3 "search $PACKAGE@$REPONAME"
-				return 1	
+				exit 1
```

Comment:
When installing multiple packages at the same time, or when upgrading, I think this causes pacstall to stop  instead of continuing to the next package on the list

Reply:
![image](https://user-images.githubusercontent.com/58742515/155621222-ca5b1f94-fccb-44b8-8811-07fca4f12263.png)

Reply:
Ok, then error out on install. But what about on upgrade tho?

Reply:
That should be fixed with #503